src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    static Task[] taskList = new Task[100];
+    static Boolean[] taskStatusList = new Boolean [100];
+    static int currentCount = 0;
+    static String dashedLine = "\t____________________________________________________________";
```

Comment:
I think constants should be named using all cap and there should be a final before to mark as constant.